templates/land-see/partials/image.snip.html

```diff
@@ -0,0 +1,40 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#image}}
```

Comment:
This use-case does not warrant a partial , i would say just inline the image tag in the template and fill in with mustache.

Reply:
I was planning on consuming the partial in many of the post objects that will come in a future PR. That single line partial reference `{{> ./image.snip.html}}` in that markup has made things much more readable. This is a setup for those commits. As mentioned, [here](https://github.com/ampproject/ampstart/issues/602), however, that kind of thinking has gotten me in trouble with some nested amp components.

Where it really helps reduce markup, this has been my approach in my feature branches. I can certainly remove going forward, however.